proto/buf/alpha/registry/v1alpha1/reference.proto

```diff
@@ -41,6 +43,13 @@ message RepositoryMainReference {
   RepositoryCommit commit = 2;
 }
 
+message RepositoryDraft {
+  // The name of the draft
+  string name = 1;
+  // The commit this draft points to.
+  RepositoryCommit commit = 2;
```

Comment:
Just for my understanding - there can be any number of drafts in a repository, but a draft always corresponds to exactly one commit?

Reply:
Correct. And when a user would push a new commit with the same draft name, it'll first remove any existing commit with that name if one exists.